packages/forms/src/UIForm/fields/RadioOrSelect/RadioOrSelect.component.js

```diff
@@ -14,7 +14,7 @@ export default function RadioOrSelect(props) {
 
 if (process.env.NODE_ENV !== 'production') {
 	RadioOrSelect.propTypes = {
-		schema: PropTypes.shape({ titleMap: PropTypes.array }).isRequired,
+		schema: PropTypes.shape({ titleMap: PropTypes.array }),
```

Comment:
To avoid breaking changes, everywhere there is a default prop that is used in the component, we let it. So to fix the lint error where a prop is required but a default prop is provided, we remove the `required` in proptypes.